Update out of date analytics.js docs (#6420)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/index.md

## External dependencies

> Analytics.js production dependencies can be found [listed under the **dependencies** key](https://github.com/segmentio/analytics-next/blob/master/packages/browser/package.json){:target="_blank”}.

Reviewer: ```suggestion
Analytics.js production dependencies are [listed under the **dependencies** key](https://github.com/segmentio/analytics-next/blob/master/packages/browser/package.json){:target="_blank”}.
```
